src/neo-vm/ExecutionContext.cs

```diff
@@ -1,11 +1,15 @@
-﻿using System.Diagnostics;
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
 using System.Runtime.CompilerServices;
 
 namespace Neo.VM
 {
     [DebuggerDisplay("RVCount={RVCount}, InstructionPointer={InstructionPointer}")]
     public class ExecutionContext
     {
+        private readonly Dictionary<Type, object> states = new Dictionary<Type, object>();
```

Comment:
Maybe is better to use this https://docs.microsoft.com/en-us/dotnet/api/system.dynamic.expandoobject?view=netframework-4.8

Reply:
The performance of `ExpandoObject` is very poor.

Reply:
I think that Map should resolve this for us Shargon... if performance is not the best, in time we improve it. Best thing now is to keep things moving.